include/lsst/afw/table/Exposure.h

```diff
@@ -1,8 +1,8 @@
 // -*- lsst-c++ -*-
-/* 
+/*
```

Comment:
Ah, I meant to revert the whitespace changes before I pushed.  I might still do that with a force push, just to keep the changes distinct.  I've been trying out Atom as my text editor, and it actually just goes ahead and *deletes* all trailing whitespace automatically, which is a bit overzealous, especially when making small changes to old files.